docs/extraction_quickstart.rst

```diff
@@ -0,0 +1,93 @@
+.. _extraction_quickstart:
+
+Spectral Extraction Quick Start
+===============================
+
+Specreduce provides flexible functionality for extracting a 1D spectrum from a 2D spectral image,
+including steps for determining the trace of a spectrum, background subtraction, and extraction.
+
+
+Tracing
+-------
+
+The `specreduce.tracing` package allows for defining the trace of a spectrum on the 2D image.  These
```

Comment:
1. I prefer "defines the trace" or "allows users to define the trace" over "allows for defining the trace." If you change it here, the "Background" section needs it too. ("Extraction" already uses the first construction.)
2. Should "package" be "module" here since `tracing.py` is a file? If so, the other sections need the same change.